Makefile

```diff
@@ -567,11 +569,13 @@ openstack_prep: export IMAGE=${OPENSTACK_IMG}
 openstack_prep: $(if $(findstring true,$(BMO_SETUP)), crc_bmo_setup) ## creates the files to install the operator using olm
```

Comment:
Looks like sometimes the order is no honoured, we probably have to add the prereqs with `\` in one line.

Reply:
Sorry I did not get this part, you mean `openstack_prep: $(if $(findstring true,$(BMO_SETUP)), crc_bmo_setup) \` or something else? An example would be helpful.

Reply:
do something like below and it should work...

```
+OPENSTACK_PREP_DEPS := $(if $(findstring true,$(INSTALL_CERT_MANAGER)), certmanager)
+OPENSTACK_PREP_DEPS := $(if $(findstring true,$(NETWORK_ISOLATION)), nmstate nncp netattach metallb metallb_config)
+OPENSTACK_PREP_DEPS += $(if $(findstring true,$(NETWORK_BGP)), nmstate nncp netattach metallb metallb_config)
+OPENSTACK_PREP_DEPS += $(if $(findstring true,$(BMO_SETUP)), crc_bmo_setup) ## creates the files to install the operator using olm
+
 ##@ OPENSTACK
 .PHONY: openstack_prep
 openstack_prep: export IMAGE=${OPENSTACK_IMG}
-openstack_prep: $(if $(findstring true,$(BMO_SETUP)), crc_bmo_setup) ## creates the files to install the operator using olm
-openstack_prep: $(if $(findstring true,$(NETWORK_BGP)), nmstate nncp netattach metallb metallb_config)
-openstack_prep: $(if $(findstring true,$(NETWORK_ISOLATION)), nmstate nncp netattach metallb metallb_config)
+openstack_prep: $(OPENSTACK_PREP_DEPS)
        $(eval $(call vars,$@,openstack))
        bash scripts/gen-olm.sh
 
 .PHONY: openstack
-openstack: certmanager operator_namespace openstack_prep ## installs the operator, also runs the prep step. Set OPENSTACK_IMG for custom image.
+openstack: operator_namespace openstack_prep ## installs the operator, also runs the prep step. Set OPENSTACK_IMG for custom image.
        $(eval $(call vars,$@,openstack))
        oc apply -f ${OPERATOR_DIR}
```